phpunit/fonts-library/wpFonFamilyUtils/getFilenameFromFontFace-test.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Test WP_Font_Family_Utils::get_filename_from_font_face().
+ *
+ * @package WordPress
+ * @subpackage Fonts Library
+ *
+ * @group fonts
+ * @group fonts-library
+ *
+ * @covers WP_Font_Family_Utils::get_filename_from_font_face
+ */
+class Tests_Fonts_WpFontsFamilyUtils_GetFilenameFromFontFace extends WP_UnitTestCase {
```

Comment:
```suggestion
class Tests_Fonts_WpFontFamilyUtils_GetFilenameFromFontFace extends WP_UnitTestCase {
```
Minor typo in the class name

Reply:
Done https://github.com/WordPress/gutenberg/pull/53702/commits/8145df112ca0469dc3a45c6ce9c1ede9389a3b12 ✅